tinygrad/graph.py

```diff
@@ -4,12 +4,14 @@
 except ImportError:
   nx = None # graph won't work
 from collections import defaultdict
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, TYPE_CHECKING
 from tinygrad.ops import UnaryOps, BinaryOps, ReduceOps, MovementOps, LoadOps, TernaryOps, Op, OpType, LazyOp
-from tinygrad.tensor import LazyBuffer
 from tinygrad.helpers import GRAPH, GRAPHPATH, PRUNEGRAPH, DEBUG, GlobalCounters
 from tinygrad.runtime.lib import RawConst
 
+if TYPE_CHECKING:
+  from tinygrad.lazy import LazyBuffer
```

Comment:
Can this be single lined? or can you not put an import after an if on a single line?

Reply:
Sure. Same thing in `ops.py`, single-lined it there as well.